README.md

```diff
@@ -179,13 +179,101 @@ All the widgets will have the same size, which is chosen based on the number of
 Custom metric monitoring can be created for simple metrics, simple metrics with anomaly detection and search metrics.
 The first two also support alarming.
 
-### Custom monitoring
+#### Example: simple metrics
+
+```ts
+monitorCustom({
+    title: "Three simple metrics",
+    metrics: [m1, m2, m3]
```

Comment:
It'd probably help to show a sample of a metric, or just add a comment stating what sort of metric classes/interfaces are allowed.

Reply:
I see, so you mean for m1, etc?

Reply:
Yeah. A comment describing the allowed types should be sufficient.